package.json

```diff
@@ -140,7 +140,7 @@
         "@0x/contract-wrappers": "^13.15.0",
         "@0x/contracts-zero-ex": "^0.21.1",
         "@0x/json-schemas": "^5.3.1",
-        "@0x/mesh-graphql-client": "^11.0.1",
+        "@0x/mesh-graphql-client": "^11.2.0",
```

Comment:
The `yarn.lock` file didn't seem to update, try doing a `yarn install` (possibly `yarn install --force`) and updates the file.

Reply:
🙏